packages/components/src/components/switch/switch.tsx

```diff
@@ -33,7 +33,7 @@ export class Switch {
   @Element() public host!: HTMLElement;
 
   /** Aligns the label. */
-  @Prop() public alignLabel?: BreakpointCustomizable<SwitchAlignLabel> = 'right';
+  @Prop() public alignLabel?: BreakpointCustomizable<SwitchAlignLabel> = 'end';
```

Comment:
No deprecation console log or anything here?

Reply:
As far as I know, we are not able to provide a deprecation warning for breakpoint customizable prop values.